### DISCUSSION TITLE
Breezy Weather new icon set

### DISCUSSION THREAD

**Discussion by whitewallman** [breezy-weather/breezy-weather#1388] [General]
Don't know if this is the proper place to share this, but there's a new Google style icon set compatible with Breezy Weather.
https://play.google.com/store/apps/details?id=com.kole.materialweather
![Screenshot_20241030-151746.png](https://github.com/user-attachments/assets/889e1f2c-7e8e-4b47-b593-78dfbb520522)

**Comment by mbatthew** [comment]
Hey, thanks for sharing! I didn’t realize some Komponent apps could work with Breezy Weather.

I actually saw [Delta Icons’ pack](https://github.com/Delta-Icons/breezy-weather) a while back and it inspired me to make something similar to what you posted — a dedicated icon pack app for Breezy Weather using Google’s Weather Icons v4.

Similar to @kole’s excellent Komponent setup, mine’s a direct icon pack integration (just install and select in Breezy Weather).

**Post & details:** https://github.com/breezy-weather/breezy-weather/discussions/2110
**Download:** https://github.com/mbatthew/GoogleWeatherIconsV4Pack/releases/tag/v1.0.0

**Reply by whitewallman** [reply]
Awesome! Thank you so much!
